README.md

```diff
@@ -25,6 +25,12 @@ With MembraneCurvature you can:
 Installation
 --------------
 
+MembraneCurvature is available via pip:
+
```

Comment:
It might be useful to say that it's main dependency is MDAnalysis, which can be installed with conda (and link to the MDAnalysis installation instructions in the User Guide). In this way, `pip` won't pull in a million dependencies that many people would rather handle with conda.

Reply:
I think powered by MDA is probably enough, yeah,